docs/getting-started.md

```diff
@@ -364,6 +364,23 @@ not run on an EC2 .metal instance. You can skip performance tests with:
  ./tools/devtool test -- --ignore integration_tests/performance
  ```
 
+## Troubleshooting
+
+#### Errors while using `curl` to access the API
+
+Points to check to confirm the API socket is running and accessible:
+* Check that the user running the Firecracker process and the user using `curl` have equivalent privileges. 
+  For example, if you run Firecracker with **sudo** that you run `curl` with **sudo** as well
+* **SELinux** can regulate access to sockets on RHEL based distributions. 
+  You can check the configuration of **SELinux** in `/etc/selinux/config`.
+* With the Firecracker process running using `--api-sock /tmp/firecracker.socket`, confirm that the socket is open:
+  * `netstat -a | grep '/tmp/firecracker.socket'`
```

Comment:
`netstat` is deprecated in favor of `ss`. It has similar options so `ss -a` should work.